### PR TITLE
Configure publishing for Android AAR via Maven

### PR DESCRIPTION
**Pull request by AMITGOELNY**
### _Summary_
Configure publishing for Android AAR via Maven.  

* Updates gradle wrapper to 7.3
* Allows Native Android SDK to be distributed via MavenCentral

### _How did you test your changes?_
manually

### CODE REVIEW

File: packages/wallet-native-sdk/android/CoinbaseWalletNativeSDK/build.gradle [hunk 1]
```diff
@@ -44,4 +53,59 @@ dependencies {
...
+                    description = 'Coinbase NativeWallet SDK'
+                    url = 'https://github.com/coinbase/coinbase-wallet-sdk'
+                    licenses {
+                        license {
+                            name = 'Coinbase License'
+                            url = 'https://github.com/coinbase/coinbase-wallet-sdk/blob/master/LICENSE'
+                        }
+                    }
+                    developers {
+                        developer {
```
[the remark this answers is not in the document]
Comment (bangtoven) [reply packages/wallet-native-sdk/android/CoinbaseWalletNativeSDK/build.gradle:78]: mine looks good. we should get vishnu's email

File: packages/wallet-native-sdk/android/CoinbaseWalletNativeSDK/build.gradle [hunk 2]
```diff
@@ -44,4 +53,59 @@ dependencies {
...
+        publications {
+            release(MavenPublication) {
+                from components.release
+                groupId 'com.coinbase'
+                artifactId 'coinbase-wallet-sdk'
+                version = android.defaultConfig.versionName
+
+                pom {
+                    name = 'coinbase-wallet-sdk'
+                    description = 'Coinbase NativeWallet SDK'
```
Comment (bangtoven) [inline packages/wallet-native-sdk/android/CoinbaseWalletNativeSDK/build.gradle:69]:
```suggestion
                    description = 'Coinbase Native Wallet SDK'
```

**Review by bangtoven — COMMENTED** [review]
yay!

**Review by bangtoven — APPROVED** [review]

**Review by vishnumad — APPROVED** [review]

**Comment by Vladisvob** [comment]
#602

**Review by Vladisvob — COMMENTED** [review]
@Vladisvob

File: packages/wallet-native-sdk/android/CoinbaseWalletNativeSDK/build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vladisvob) [reply packages/wallet-native-sdk/android/CoinbaseWalletNativeSDK/build.gradle:78]: @Vladisvob

**Review by Vladisvob — COMMENTED** [review]
#620

**Comment by Vladisvob** [comment]
@dependabot rebase

**Pull request closed without merging**